.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,75 @@
+  - type: markdown
+    attributes:
+      value: |
+        ### Thanks for helping Conda! 🐍
+
+        Going through feature requests and issues takes up a lot of time, so please be so kind and take a few minutes to fill out all the areas to the best of your ability.
+         
+        There will always be more great ideas than there is time to do them, and so we will need to selectively close issues that don't provide enough information, so we can focus our time on helping people like you who fill out the form completely. Thank you for your collaboration!
+        
+        There are also already a lot of open requests, so please take 2 minutes and search through existing ones to see if your idea already exists. If you find something close, please upvote that request and comment
+
+        Thanks for helping Conda be amazing. We are nothing without people like you helping build a better community 💐!
+
+        ### Lets make sure you are in the right place. If you have an idea/request for:
+
+        - ####  A specific conda package from Anaconda ('defaults' channel), or  [repo.anaconda.com](repo.anaconda.com) access and service::
+
+          &rarr; file it at [anaconda issues](https://github.com/ContinuumIO/anaconda-issues)
+
+        - #### A specific conda package from conda-forge:
+
+          &rarr; file at the corresponding feedstock under [conda-forge](https://github.com/conda-forge)
+          
+        - #### anaconda.org access and service:
+
+          &rarr; file it [here](https://anaconda.org/contact/report)
+
+        - #### Commands under 'conda build':
+
+          &rarr; file it at [conda-build](https://github.com/conda/conda-build)
+
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: Please confirm and check all the following options
+      options:
+        - label: I added a descriptive title
+          required: true
+        - label: I searched for other feature request and couldn't find a duplication (including also the ``type-feature`` tag)
+          required: true
+        - label: I confirmed that its not related to another project (see the above section)
+          required: true
+  - type: textarea
+    id: request-idea
+    attributes:
+      label: What is the idea?
+      description: Describe what the feature is and the desired state
+      placeholder: This feature would allow any user of Conda to type in a simple command and be presented with an extra field next to the package showing what license it uses
+    validations:
+      required: true
+  - type: textarea
+    id: why
+    attributes:
+      label: Why is this needed
+      description: |
+        Who would benefit from this and why would this add value to them? What problem does this solve?
+      placeholder: This would benefit a user that would like to see what licenses they are using. This solves the problem where someone needs to know that they don't have any restrictive licenses in their codebase
```

Comment:
```suggestion
      placeholder: This would benefit users who would like to see what licenses they are using. This solves the problem where someone needs to know that they don't have any restrictive licenses in their codebase
```